src/service/command/executor/command-executor-delegator.ts

```diff
@@ -0,0 +1,55 @@
+import { Service } from "typedi";
+import { BashExecutor } from "@bc/service/command/executor/bash-executor";
+import { ExportExecutor } from "@bc/service/command/executor/export-executor";
+import { ExecuteCommandResult, ExecutionResult } from "@bc/domain/execute-command-result";
+import { LoggerServiceFactory } from "@bc/service/logger/logger-service-factory";
+
+@Service()
+export class CommandExecutorDelegator {
+
+  private _bashExecutor: BashExecutor;
+  private _exportExecutor: ExportExecutor;
+
+  constructor(bashExecutor: BashExecutor,
+              exportExecutor: ExportExecutor) {
+    this._bashExecutor = bashExecutor;
+    this._exportExecutor = exportExecutor;
+  }
```

Comment:
decorate instance variables with `@inject` instead of initializing in constructor?